desc/objectives/objective_funs.py

```diff
@@ -33,21 +34,38 @@ class ObjectiveFunction(IOAble):
         Method for computing Jacobian matrices. "batched" uses forward mode, applied to
         the entire objective at once, and is generally the fastest for vector valued
         objectives, though most memory intensive. "blocked" builds the Jacobian for each
-        objective separately, using each objective's preferred AD mode. Generally the
-        most efficient option when mixing scalar and vector valued objectives.
-        "looped" uses forward mode jacobian vector products in a loop to build the
-        Jacobian column by column. Generally the slowest, but most memory efficient.
+        objective separately, using each objective's preferred AD mode (and
+        each objective's `chunk_size`). Generally the most efficient option when mixing
+        scalar and vector valued objectives. "looped" uses forward mode jacobian vector
+        products in a loop to build the Jacobian column by column. Generally the
+        slowest, but most memory efficient.
         "auto" defaults to "batched" if all sub-objectives are set to "fwd",
```

Comment:
I think I will keep as is: if you give chunk_size to sub obj, it will warn that it wont be used unless you do blocked, and make them request it specifically